.gitmodules

```diff
@@ -59,3 +59,6 @@
     branch = master
 
 # Note: do not add new plugin submodules here, but a few lines above
+[submodule "plugins/Morpheus/icons"]
```

Comment:
fixed in https://github.com/piwik/piwik/pull/11383/commits/af4324fe7503a4303aa81bc8c9a706ecd55a25a8